configs/components/openssl.rb

```diff
@@ -54,6 +66,23 @@
     ldflags = ''
     cflags = "$${CFLAGS} -static-libgcc"
     pkg.environment "CC" => "/opt/pl-build-tools/bin/gcc"
+  elsif platform.is_windows?
+    if platform.architecture == "x64"
+      target = "mingw64"
+      arch = "64"
+    else
+      target = "mingw"
+      arch = "32"
+    end
```

Comment:
No reason. It just seemed more legible at the time. Do you think it would be more logical to have these be ternaries?